lib/options.json

```diff
@@ -436,7 +447,7 @@
       "noInfo": "should be {Boolean} (https://webpack.js.org/configuration/dev-server/#devservernoinfo-)",
       "onListening": "should be {Function} (https://webpack.js.org/configuration/dev-server/#onlistening)",
       "open": "should be {String|Boolean} (https://webpack.js.org/configuration/dev-server/#devserveropen)",
-      "openPage": "should be {String} (https://webpack.js.org/configuration/dev-server/#devserveropenpage)",
+      "openPage": "should be {String|Array} (https://webpack.js.org/configuration/dev-server/#devserveropenpage)",
```

Comment:
We have to update the documentation.

Reply:
I have opened a PR.
https://github.com/webpack/webpack.js.org/pull/3306